example/android/build.gradle

```diff
@@ -1 +1,10 @@
-// Top-level build file where you can add configuration options common to all sub-projects/modules.
+buildscript {
```

Comment:
Why do we also need this file now?

Reply:
Related to the comment above. ADP plugin has to be applied at the project root level (if you have more than one module, app + support). The project root is where your `settings.gradle` file is declared. If you want to be able to develop `RNTA/android` and `RNTA/example/android` in AS independently, they have to be distinct projects.